src/utils/pickNumberInList.js

```diff
@@ -1,8 +1,6 @@
-import CONFIG from '../constants/config.js';
-
-export default function pickNumberInList(min, max) {
+export default function pickNumberInList({ min, max, length }) {
   const randomNumbers = new Set();
-  while (randomNumbers.size < CONFIG.LOTTO.LENGTH) {
+  while (randomNumbers.size < length) {
```

Comment:
최소 범위에 대한 기준이 필요할 수도 있으니

```suggestion
  while (randomNumbers.size < maxLength) {
```

Reply:
`length`보다 `maxLength` 로 표현하면 최소 범위에 대한 표현이라는 게 명확해지네요
조건문 내에 변수를 쓸 때 이런 부분에 대해 앞으로 더 신경 써야겠습니다